packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -9,6 +19,12 @@ import onErrorCallback from 'utils/onErrorCallback';
 import { defaultMessages, Messages } from './messages';
 import { formatMessage } from 'utils/formatMessage';
 
+/**
+ * A pagination component to allow users to navigate through multiple pages of results
+ *
+ * Returns a reset function via the `resetRef` prop that resets the component to page 1.
```

Comment:
This isn't true. It returns a component, but we externalize an API through use of `resetRef`.

Also, now that I think of it, this ref shouldn't be called `resetRef` since we could conceivably add other external API methods to that. Also, it leads to usage like:

```
<ResultsPagination resetRef={resetRef} ... />

...

resetRef.current.reset();
```

Double "reset"!

I don't think we need to be fancy here -- we can just use `ref`. Also, that'll clean up the code at the end of the file to just be:

```
export default forwardRef(ResultsPagination);
```

(But with TS)

Reply:
Agree that we can just go with `ref`. The end piece still needs to be a bit complex due to using an HOC and needing refs to be passed in.

Reply:
Unfortunately, this _did_ have to renamed to something besides `ref`: https://dev.to/justincy/using-react-forwardref-and-an-hoc-on-the-same-component-455m

I'll create a PR to revert this-- you can let me know what name suits you. 😄